helm/operator/values.yaml

```diff
@@ -280,6 +280,7 @@ console:
   # Configures `Ingress <https://kubernetes.io/docs/concepts/services-networking/ingress/>`__ for the Operator Console.
   #
   # Set the keys to conform to the Ingress controller and configuration of your choice.
+  # set ingress.number to 9443 for HTTPS and 9090 for HTTP as desired
```

Comment:
Are these ports hard-coded, or will it ultimately depend on whether ingress / MinIO backend is configured for TLS?

e.g. lets say I configure MinIO to listen on `81` and console to listen on `80` for some reason. Would all that matters be that the ports match up below?

Reply:
This is not for the tenant at all. It is for the operator console only. And they should never be hardcoded. It is just a comment and any port is more than welcome. Will rephrase it and add MINIO_CONSOLE_TLS_ENABLE as suggested by Jiuker

Reply:
Hey, but thanks for the review :)